LICENSE

```diff
@@ -1,2 +1,2 @@
-YEAR: 2022
-COPYRIGHT HOLDER: Ascent Digital Services UK Limited
```

Comment:
can we just change them?

Reply:
Oh yeah, good point, sorry I was too rushed there, but of course we can not change that. It's just an agreement to transfer maintenance not copyright. So good to have your observant :eyes: !